target/linux/ipq40xx/image/Makefile

```diff
@@ -257,6 +257,24 @@ define Device/engenius_ens620ext
 endef
 TARGET_DEVICES += engenius_ens620ext
 
+define Device/ezviz_cs-w3-wd1200g-eup
+	$(call Device/FitImage)
+	DEVICE_VENDOR := EZVIZ
+	DEVICE_MODEL := CS-W3-WD1200G
+	DEVICE_VARIANT := EUP
+	BOARD_NAME := cs-w3-wd1200g-eup
+	DEVICE_DTS_CONFIG := config@4
+	KERNEL_SIZE := 4096k
```

Comment:
Is this the kernel size from the vendor firmware? It's not used anywhere, so it could be dropped.

Reply:
Your are correct. I've removed the the KERNEL_SIZE variable